stl/inc/future

```diff
@@ -307,25 +334,11 @@ public:
 
     void _Set_value_raw(_Ty&& _Val, unique_lock<mutex>* _Lock,
         bool _At_thread_exit) { // store a result while inside a locked block
-        if (_Has_stored_result) {
-            _Throw_future_error(make_error_code(future_errc::promise_already_satisfied));
-        }
-
-        _Result = _STD forward<_Ty>(_Val);
-        _Do_notify(_Lock, _At_thread_exit);
-    }
-
-    void _Set_value(bool _At_thread_exit) { // store a (void) result
-        unique_lock<mutex> _Lock(_Mtx);
-        _Set_value_raw(&_Lock, _At_thread_exit);
-    }
-
-    void _Set_value_raw(
-        unique_lock<mutex>* _Lock, bool _At_thread_exit) { // store a (void) result while inside a locked block

```

Comment:
I see that these are being removed because they were completely unused. That's fine (they're non-virtual and there are no downsides to deleting them), but I recommend editing your PR description to mention this removal.